src/ostorlab/agent/message/proto/v3/report/cve/cve.proto

```diff
@@ -0,0 +1,111 @@
+syntax = "proto2";
+
+package ostorlab.agent.message.proto.v3.report.cve;
+
+message Message {
+    optional string cve_id = 1;
+    repeated Target targets = 2;
+    optional int32 cwe = 3;
+    optional string description = 4;
+    repeated Reference references = 5;
+    optional CvssScores cvss_scores = 6;
+    optional uint64 published_date = 7;
+    optional uint64 modified_date = 8;
+}
+
+message CvssScores { 
+    optional CvssScoreV3 cvss_v3_1 = 1;
+    optional CvssScoreV3 cvss_v3_0 = 2;
+    optional CvssScoreV2 cvss_v2 = 3;
+}
```

Comment:
```suggestion
message CVSS { 
    optional CVSSv31 v31 = 1;
    optional CVSSv30 v30 = 2;
    optional CVSSv2 v2 = 3;
}
```

Reply:
Do we really need both CVSSV3.1 and CVSSV3.0 versions or keep only the latest one since we need to persist with only one?
And what if other versions occur? CVSS v4.0, for example, that could be found here : https://www.first.org/cvss/v4-0/index.html

Reply:
We need all.